nest/03.nest.md

```diff
@@ -0,0 +1,89 @@
+# 2 nest
+In this chapter, we will detail how to use the binary `nest`.
+As explained before, this binary is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest options
```

Comment:
I'm not sure this section is very useful. There's not so much interest in its content, and it assumes (by it's last sentence about how to form a valid command) that the user haven't typed any command before.

While it's great to start slowly, maybe that's a bit too slow :p We can assume the user has a tiny bit of user experiance with command line and terminals before.